pysparkle/pysparkle/pysparkle_cli.py

```diff
@@ -37,8 +38,34 @@ def gold(partitions):
     gold_main(partitions)
 
 
+dq_config = {
+    "container_name": "staging",
+    "dataset_name": "movies_metadata",
```

Comment:
So currently in bronze (`raw`) container we have `movies_dataset` "folder" and inside this folder there is a number of CSV files. All these CSV files are written directly to "staging" schema. Aren't we mixing the dataset name with table names?

Reply:
Yeah, I was initially confused by naming it "dataset_name" - while you were replying I edited my initial comment to simply change this parameter name if possible:

```suggestion
    "table_name": "movies_metadata",
```

Reply:
Can do, I'd just gone with dataset as it's the GX standard language, but happy to amend this to table_name if it's more user friendly. I'd bear in mind though that we'll use the same config model regardless of raw/staging etc so I'm not sure table name will always make sense (although for the same reason dataset_name might not either

Reply:
Ok for me to rename the other one to `folder_name` in a separate PR?

Reply:
yes, I am! :D

Reply:
Done